common/config.go

```diff
@@ -1,61 +1,58 @@
 package common
 
 import (
-	"errors"
+	"os"
+
+	"github.com/iost-official/Go-IOS-Protocol/ilog"
 	"github.com/spf13/viper"
+	yaml "gopkg.in/yaml.v2"
 )
 
+type DBConfig struct {
+	LdbPath string
+}
+
+type VMConfig struct {
+}
+
+type P2PConfig struct {
+	Address string
+	Port    int64
+}
+
+// Config provide all configuration for the application
 type Config struct {
-	vip *viper.Viper
-
-	LdbPath       string
-	RedisAddr     string
-	RedisPort     int64
-	NetLogPath    string
-	NodeTablePath string
-	NodeID        string
-	ListenAddr    string
-	RegAddr       string
-	RpcPort       string
-	Target        string
-	Port          int64
-	MetricsPort   string
-	AccSecKey     string
-	LogPath       string
-	CfgFile       string
-	LogFile       string
-	DbFile        string
+	VM  VMConfig
```

Comment:
You should use pointer instead of value since the config may inflate in the future.

Reply:
Thx, I'll do it.